### PR TITLE
Improves Sentry logging on the keepalive endpoint

### PR DESCRIPTION
**Pull request by asg5704**
## Description
**Issue** [Sentry SSOe Error Cleanup](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16388)
This PR adds logic to the catch statement for the `keepAlive` method in `keepAliveSSO.js` .  This will reduce the amount of non-SSOe Errors that are being captured in Sentry (ie. Things like Internet or network connection being lost) and instead use Sentry's built-in LogLevels.

## Testing done
- Manual

## Screenshots
n/a

## Acceptance criteria
- [x] Research Sentry `setlevel` & Sentry JavaScript documentation
- [x] Classify the correct error levels for `SSOe errors` to ensure appropriate logging in Sentry
- [x] Provide a default log level for `SSOe errors` for Sentry
- [x] Errors will have more context to ensure the Identity team is able to properly resolve issues discovered via Sentry

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/utilities/sso/keepAliveSSO.js [hunk 1]
```diff
@@ -34,9 +34,30 @@ export default async function keepAlive() {
...
+        logType: Sentry.Severity.Info,
+      },
+    }[err.message];
+
     Sentry.withScope(scope => {
       scope.setExtra('error', err);
-      Sentry.captureMessage(`SSOe error: ${err.message}`);
+      Sentry.captureMessage(
+        `SSOe error: ${err.message}`,
+        caughtExceptions ? caughtExceptions.logType : err.message,
```
Comment (ericbuckley) [inline src/platform/utilities/sso/keepAliveSSO.js:59]:
```suggestion
        caughtExceptions ? caughtExceptions.logType : Sentry.Severity.Error,
```

File: src/platform/utilities/sso/keepAliveSSO.js [hunk 2]
```diff
@@ -34,9 +34,30 @@ export default async function keepAlive() {
       }[resp.headers.get('va_eauth_csid')],
     };
   } catch (err) {
+    const caughtExceptions = {
```
Comment (ericbuckley) [inline src/platform/utilities/sso/keepAliveSSO.js:37]: @AGarcia-Clarity can we make the mapping a const on the module, so its just initialized once and not every time the catch block executes?

**Comment by AGarcia-Clarity** [comment]
@department-of-veterans-affairs/frontend-review-group Any ideas on getting rid of the Sentry linting issues?

**Review by joeniquette — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**